morf-core/src/main/java/org/alfasoftware/morf/metadata/Schema.java

```diff
@@ -17,13 +17,15 @@
 
 import java.util.Collection;
 
+import org.alfasoftware.morf.jdbc.AdditionalMetadata;
+
 
 /**
  * Provides database schema meta data.
  *
  * @author Copyright (c) Alfa Financial Software 2009
  */
-public interface Schema {
+public interface Schema extends AdditionalMetadata {

```

Comment:
Schema should not extend AdditionalMetadata.
That would be a far-reaching change, exposing the AdditionalMetadata to all schema consumers.
AdditionalMetadata should sit next to Schema, on MetadataProviders.
And ideally remain fairly hidden :)

Reply:
Switched it round so AdditionalMetadata extends Schema.